src/app/models/manuscript.ts

```diff
@@ -8,12 +8,23 @@ export interface KeywordGroups {
   };
 }
 
+export interface Person {
+  readonly _id: string;
```

Comment:
As per my previous comment, this will need to be writable at some point.

Reply:
If there're author IDs in XML we can still make it read-only as it would be set when an object is created from XML.